### PR TITLE
Generic component

### PR DESCRIPTION
**Pull request by benjaminxu251**

### CODE REVIEW

**Review by eneadodi — COMMENTED** [review]
Overall it looks good. However I would like you to switch the current editor with the Suneditor we have decided to use before approval

File: Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/scenario_components/InformationItem.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import SunEditor from "suneditor-react";
+import 'suneditor/dist/css/suneditor.min.css';
+import Button from '@material-ui/core/Button';
+
+export default function InformationItem() {
+  return (
+    <div>
+      <SunEditor
```
Comment (powellsl99) [inline Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/scenario_components/InformationItem.js:29]: You should look into where @Enoch-Hsiao implemented this in his PR, you can just copy paste a few things. Specifically he got it more intractable on mobile. Maybe that should be be put in its own file with the settings and then imported across the app so you dont have to keep copy pasting!

**Review by powellsl99 — CHANGES_REQUESTED** [review]
Looks great! Just one comment about sun editor

**Comment by Enoch-Hsiao** [comment]
Reopen generic component pr

File: Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/scenario_components/InformationItemList.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  handleDelete = (iItemID) => {
+    const iItems = this.state.iItems.filter((q) => q.id !== iItemID);
+    this.setState({ iItems });
+  };
+
+  updateIItem = (iItemID, iItemBody) => {
+    const newIItems = [...this.state.iItems];
+    
+    //does this even save stuff to the backend?
+    newIItems.forEach(element => {if(element.props.id === iItemID){element.props.body = iItemBody}})
```
Comment (Enoch-Hsiao) [inline Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/scenario_components/InformationItemList.js]: Not yet, but this is where we will put the logic to save data to backend once we set it up.

File: Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/scenario_components/InformationItem.js [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+        flexDirection="row"
+        p={1}
+        m={1}
+        bgcolor="background.paper"
+      >
+      {/*
+        <Box p={1}>
+          <form noValidate autoComplete="off">
+            <TextField
+              style={{ width: 500 }}
```
Comment (Enoch-Hsiao) [inline Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/scenario_components/InformationItem.js]: Remove comments?

File: Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/scenario_components/InformationItemList.js [hunk 4]
```diff
@@ -0,0 +1,74 @@
+import React, { Component } from "react";
+import InformationItem from "./InformationItem";
+import Button from "@material-ui/core/Button";
+
+class InformationItemList extends Component {
+  constructor(props) {
```
Comment (Enoch-Hsiao) [inline Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/scenario_components/InformationItemList.js]: Change to function component if you can? If not we can do it in a separate pr.

File: Ghost_in_the_Shell/client_folder_react_app/ethisim/src/pages/homepage.js [hunk 5]
```diff
@@ -161,9 +161,11 @@ export default function Homepage() {
       <StudentAccess/>
       <EthisimIntro/>
 
+    {/*}
       <Box className={classes.copyright}>
```
Comment (Enoch-Hsiao) [inline Ghost_in_the_Shell/client_folder_react_app/ethisim/src/pages/homepage.js:163]: Why?

Comment (benjaminxu251) [reply Ghost_in_the_Shell/client_folder_react_app/ethisim/src/pages/homepage.js:163]: I commented it out back when we weren't sure about using it; I'll delete it on my branch for now

File: Ghost_in_the_Shell/client_folder_react_app/ethisim/src/pages/homepage.js [hunk 6]
```diff
@@ -161,9 +161,6 @@ export default function Homepage() {
       <StudentAccess/>
       <EthisimIntro/>
 
-      <Box className={classes.copyright}>
-        <Copyright />
-      </Box>
     </div>
```
Comment (Enoch-Hsiao) [inline Ghost_in_the_Shell/client_folder_react_app/ethisim/src/pages/homepage.js]: Sorry for being strict but keep the copyright on the homepage XD

**Review by Enoch-Hsiao — DISMISSED** [review]
Looks good, add back copyright and resolve the conflict.

File: Ghost_in_the_Shell/client_folder_react_app/ethisim/src/pages/homepage.js (hunk 6 above)
Comment (benjaminxu251) [reply Ghost_in_the_Shell/client_folder_react_app/ethisim/src/pages/homepage.js]: lmao aight

**Review by Enoch-Hsiao — DISMISSED** [review]

**Review by powellsl99 — APPROVED** [review]
LGTM!

File: Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/scenario_components/Logistics.js [hunk 7]
```diff
@@ -28,8 +30,98 @@ const useStyles = makeStyles((theme) => ({
...
+
+  const addAuthor = event => {
+    setAuthor(authors.concat(<Author />))
+    event.preventDefault()
+  }
+
+  //default if it's a browser
+  var body = (
     <div>
+<<<<<<< HEAD
```
Comment (powellsl99) [inline Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/scenario_components/Logistics.js]: Actually before you merge, can you fix this

**Review by powellsl99 — CHANGES_REQUESTED** [review]

**Review by powellsl99 — APPROVED** [review]
Looks good thanks for implementing the changes! Nice job! 
As Enoch said: mergeeee

**Pull request merged**